### ISSUE TITLE
OSX Tells me to not install this 

### ISSUE THREAD

**Issue by bradennapier** [kencrocken/FiraCodeiScript#5]
![image](https://user-images.githubusercontent.com/15365418/37262075-14917bc6-255e-11e8-9117-d8f90293322b.png)

**Comment by kencrocken** [comment]
Any more details?

**Comment by bradennapier** [comment]
Nope.  I pressed download, I opened it, it told me this, I stopped.  Safe to assume this is standard in OS X as this is a brand new computer.

**Comment by kencrocken** [comment]
That's unfortunate, not much to go on ... I didn't see any warnings.  Maybe
someone will have some more info..

**Comment by mandarvaze** [comment]
I know this is an old thread, but something similar happened to me recently.

I downloaded and **saved** the font to `/tmp`. It is mapped to `/private/tmp` (I think this is a default - I didn't explicitly do this)

When I tried to install it, I got the **exact same error** with no additional details.

Bit of a googling revealed that macOS likes only certain "places" from where to install the font(s) - or probably some places it *dislikes*

Long story short - I moved the font to `~/Downloads` and exact same font installed without any warnings and/or errors.
